src/utils/themeOptions.ts

```diff
@@ -0,0 +1,7 @@
+type ThemeOptions = { value: string, label: string }
```

Comment:
Personally I wouldn't define this type, it's being inferred already and you're not exporting it.

This file works just fine, but As a metter of interest, [here's a typescript playground](https://www.typescriptlang.org/play?#code/FAehAIGUBcEMCcAmBTAbgLmNAngB2eACoAWyAtsgPK7QCWA9gHYDO4AvOAN7iqwA2AV2TpwzaPFqMA5gBpwfWACNkfEWInTwAX2DBkAD1z140cAGMmY8NFIVqdSyJLkqNBiwDaAXXbgPwcC4efiERAHJYAWh6MLkFZVVwMIBBKJjtGQCg3kFhJL5aKWJoWPklFXCAGULisIys7hzQpMQEAGtS+IqkgBF2uq1Mr10wcABVZlgpAgNYMlw+YV0LFlNmZGhnCl8ACkZkAHctvPVJKQBKNgA+LizAleZ6RYA6PnopHbvA75sXe3dmM8AGaSRA7Ha-CiXG6Q5Cvcp8dhsDj7I62ZDneEJcCjAA8AFp8eBiLBUARouABIwUCD9ohzKQzG1wLBqcTWYhFizwIpYPTYcFcsTkPA4V9zgBuYA6UAQQgACQAkpAoPLKGNKj1wABRABKusousw6026J2YVwkg6kt0svAUlFeFo+gqwAeplhADUQshWBwPAAiSLRANyAMFIrQUPgAOteBtAM+WCsd0SwJ4ok2WisZjEegHVi-dYs0XWUjgJh8bAVtyWOTiat5skaKRlggAAy9PuY7eseHJ9FEyHJ5cQtCBQJFyEYZgIB1IjHA2HoAnAorI9DJ4Fo0GeWH7RHR3qFHBw+HoQLbFGPQmYHkYAjIyngXilbssHvRlQRzBEuuQFhILixw3sgcinNINwcJwXzBvQIgBqkIaZN88g1NACHVJGoZfHGbQIX08YBtK76rPauD6HwkAbMcuyoscThHj60K3KhDxPHCbwfLC34JHe9Hol4ko4hABJEgA6gQZisvaAgIKy0DDmW2bbqwNKSMg9KkKWyjSQIxYHAQbSMPm4CGWEpaVtW-BvIZ-KDrgyasLA9r0PQ-LooKQh7jKowkCpiD0L64AivAxhyByLJ8I8Zm0HwiKLKYy6rvpBCSIp8XZtO6x7lIFFUTRZoWlaYSSkAA) showing an alternate approach.

If you prefer videos, [here's a good one](https://youtu.be/Anu8vHXsavo) on a similar topic.